packages/@react-spectrum/table/src/Table.tsx

```diff
@@ -80,7 +88,14 @@ function Table<T extends object>(props: SpectrumTableProps<T>, ref: DOMRef<HTMLD
       : DEFAULT_HEADER_HEIGHT[scale],
     estimatedHeadingHeight: props.overflowMode === 'wrap'
       ? DEFAULT_HEADER_HEIGHT[scale]
-      : null
+      : null,
+    getDefaultWidth: ({width, hideHeader, isSelectionCell}) => {
+      if (isNaN(width) && hideHeader) {
```

Comment:
I still don't understand the need for the check on width, what is it for?

Reply:
Ah, good point! This made more sense in the previous implementation, in case a width was specified. Will remove this